.github/workflows/test.yml

```diff
@@ -0,0 +1,16 @@
+name: Test
+
+on: [push]
```

Comment:
Hmm, I may have merged too early. We probably want this to run on PR, not push to master? @nabeel-